database/migrate/migrations/00012_drop_unique_l1_block_number.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+-- +goose StatementBegin
+drop index if exists l1_block_number_uindex;
```

Comment:
I think we don't need to remove it. If you really want to handle reorg, you should add the complete handling logic like this:

+ soft delete from l1_block where index>=XXX
+ insert new l1_blocks
+ remember to do it inside a DB transaction

Reply:
I think adding reorg handling logic may be an overkill here because the table is only used for storing base fees now.

Reply:
added in [c83d6bd](https://github.com/scroll-tech/scroll/pull/920/commits/c83d6bd5d28cb5c925e8adc27cfbfefb29654b6e)